target/linux/ramips/dts/kn.dts

```diff
@@ -0,0 +1,112 @@
+/dts-v1/;
+
+#include "rt3050.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "KEENETIC", "ralink,rt3052-soc";
+	model = "ZyXEL Keenetic";
+
+	cfi@1f000000 {
+		compatible = "cfi-flash";
+		reg = <0x1f000000 0x800000>;
+		bank-width = <2>;
+		device-width = <2>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x30000>;
+			read-only;
+		};
+
+		partition@30000 {
+			label = "u-boot-env";
+			reg = <0x30000 0x10000>;
+			read-only;
+		};
+
+		factory: partition@40000 {
+			label = "factory";
+			reg = <0x40000 0x10000>;
+			read-only;
+		};
+
+		partition@50000 {
+			label = "firmware";
+			reg = <0x50000 0x3b0000>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "nbg-419n:green:power";
```

Comment:
```kn:green:power``` please. Some applies to all other leds as well. I'm quite sure neither your usb nor the power led in diag does work for you.